### PR TITLE
Tidy up sass and docs for breadcrumbs

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
- Removed redundant css
- Added guidance on when to use breadcrumbs
- Fixed bug on mobiles that was showing the last but one link, not the last one
- Updated style
- Changed desktop version to display left chevron before link if only one link

### How to review 
- Tests pass
- Check mobile version shows correct link

### CODE REVIEW

**Comment by jrbarnes9** [comment]
[Update breadcrumb](https://trello.com/c/wtfNwyE3/556-update-breadcrumb)

File: src/components/breadcrumb/_breadcrumb.scss [hunk 1]
```diff
@@ -31,28 +31,34 @@ $breadcrumb-chevron-height: 0.65rem;
...
     }
-
-    &--current {
-      color: $color-text-light;
-    }
   }
 
   &__link {
-    text-decoration: none;
+    color: #222;
```
Comment (jrbarnes9) [inline src/components/breadcrumb/_breadcrumb.scss:61]: Use variable: 
```suggestion
    color: $color-text;
```

File: src/components/breadcrumb/index.njk [hunk 2]
```diff
@@ -11,7 +11,8 @@ A breadcrumb trail consists of a list of links to the parent pages of the curren
 }}
 
 ## When to use this component
-Use the breadcrumbs component when you need to help users understand and move between the multiple levels of a website.
+Use the breadcrumbs component when you need to help users understand and move between the multiple levels of a website. We recomend not using it on
+pages that are only one level deep in the page hierarchy. As we would like to avoid 'home' being the only link, when it’s directly under a menu that also has ‘home’.
```
Comment (jrbarnes9) [inline src/components/breadcrumb/index.njk:15]:
```suggestion
Use the breadcrumbs component when you need to help users understand and move between the multiple levels of a website.
```
I think we'll have to remove this comment about not displaying it when home is in the nav - it's a trade-off but I think the convention and consistency of always retaining the breadcrumb is more important than the lack of value in this small case

File: src/components/breadcrumb/_breadcrumb.scss [hunk 3]
```diff
@@ -31,28 +31,34 @@ $breadcrumb-chevron-height: 0.65rem;
...
+    text-decoration: underline;
```
Comment (jrbarnes9) [inline src/components/breadcrumb/_breadcrumb.scss:62]: Removing the underline on hover is needed for a11y (shouldn't rely on colour change alone) but we could leave for a full sweep of this in: https://trello.com/c/mA0eWaT8/545-issue-803-dont-use-colour-as-the-only-indicator-of-hover-states

File: src/components/breadcrumb/_breadcrumb.scss [hunk 4]
```diff
@@ -31,28 +31,34 @@ $breadcrumb-chevron-height: 0.65rem;
         content: none;
       }
 
-      &:not(:nth-last-child(2)) {
+      &:not(:nth-last-child(1)) {
         display: inline-block;
       }
 
+      &:first-child:nth-last-child(1) {
```
Comment (jrbarnes9) [inline src/components/breadcrumb/_breadcrumb.scss:38]: This chevron needs to be left-aligned flush with edge of content block

File: src/components/breadcrumb/_breadcrumb.scss (hunk 1 above)
Comment (jrbarnes9) [reply src/components/breadcrumb/_breadcrumb.scss:61]: ...or probably a new variable for breadcrumbs?

Comment (trevorsaint) [reply src/components/breadcrumb/_breadcrumb.scss:61]: I think the breadcrumb is always going to be the text colour, so it makes sense to use the existing text variable.

**Comment by rmccar** [comment]
> Looking at the HTML It might be worth setting the font scale in the component itself rather than using utility classes for each item. We don’t really want to allow control of the breadcrumb font scale.
> 
> Also why are IDs being applied for each item?
> 
> **It might be nice adding a little negative margin to bring the alignment back. I appreciate it is only a few pixels, but its the small details :)**
> image
> 
> Negative 2px should do the trick.
> image

Ok I've fixed the margin and utility classes issues. The IDs I'll leave in. I asked, and it looks like they might be needed for craft

**Pull request merged**